integration_test/tests/common/mod.rs

```diff
@@ -14,7 +14,7 @@ pub fn generate_default_connection_str() -> String {
     let db = env::var("ADF_TEST_LOCAL_DB");
     let driver = match env::var("ADF_TEST_LOCAL_DRIVER") {
         Ok(val) => val,
-        Err(_e) => "MongoDB Atlas SQL ODBC Driver".to_string(), //Default driver name
+        Err(_e) => "MongoDB Atlas SQL ODBC DRIVER".to_string(), //Default driver name
```

Comment:
Why this change? The name I see in our reg setup script is `MongoDB Atlas SQL ODBC Driver` and Power-BI is using this one as well.
I don't know if driver names are case-sensitive, so maybe it does not matter. Still, I am curious about it ?

Reply:
Oh they absolutely are. This was probably a fat finger vim uppercase. Good catch.